modules/console/pages/flow-console.adoc

```diff
@@ -123,7 +123,11 @@ All your sites should be listed.
 . Click on a process to display the process traffic.
 +
 NOTE: The process detail displays the associated image, host, and addresses.
+You can also view the clients that are calling the process.
+
+. Click *Addresses* and choose an address to show the details for that address. This shows the set of processes that are exposed across the {service-network}.
```

Comment:
I think **servers** is better than **processes** in that case. It is more specific. 
Moreover it worth mentionate that in this view we can also see the connections (active and not active for 15 min) and requests to this address (last 15 min)